PointsRanking/Fun_PointsRanking.php

```
@@ -0,0 +1,593 @@
+function pl_points_format_number($value)
+{
+    $formatted = number_format((float) $value, 2, ',', '');
+    $formatted = rtrim($formatted, '0');
+    return rtrim($formatted, ',');
+}
+
+// --- Auto-install ------------------------------------------------------
+
+function pl_points_ensure_tables()
+{
+    $Rs = safe_r_sql("SHOW TABLES LIKE 'PLPointsTournamentConfig'");
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $Rs. Configured minimum length is 3. (undefined)

(ShortVariable)